LaunchScreenDemo/WebServices/Alamofire/Controller/JeweleryViewController.swift

```diff
@@ -0,0 +1,82 @@
+//
+//  ClothesViewController.swift
+//  LaunchScreenDemo
+//
+//  Created by Vidhi Patel on 05/07/23.
+//
+
+import UIKit
+
+protocol GetProducts: AnyObject {
+    func getProducts(products: [ProductRequestModel])
+}
+
+class JeweleryViewController: UIViewController {
+
+    // MARK: - IBOutlet and Vars
+    @IBOutlet private weak var tableView: UITableView!
+    var products: [ProductRequestModel] = []
+    private let viewModel: JeweleryViewModel = JeweleryViewModel()
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        setupUI()
+    }
+    
+    private func setupUI() {
+        navigationItem.hidesBackButton = true
+        navigationItem.title = "Women's Jewelery"
+        viewModel.delegate = self
+        viewModel.getProduct()
+        viewModel.deleteSuccess.bind { [self] in
+            viewModel.deleteProduct()
```

Comment:
you are calling API on API success why?
and I said UI should be update on delete sucess means 
 tableView.deleteRows(at: [indexPath], with: .fade)
 self.products.remove(at: indexPath.row)
these two things should be done after delete response